config/scripts/patch-ts.js

```
@@ -56,13 +56,13 @@ async function patchTypeDefs() {
 
   if (typeDefsWithCoreImports.length === 0) {
     console.log(
-      'Found no .d.ts modules containing the "~/core" import, skipping...',
+      'Found no .d.ts modules containing the "#core" import, skipping...',
     )
     return process.exit(0)
```

ESLint

[error] 'process' is not defined.

(no-undef)